terraform/modules/emrfs-lambda/policy-munge-lambda-files/lambda_handler.py

```diff
@@ -6,7 +6,8 @@
 from aws_caller import list_all_policies_in_account, get_policy_statement_as_list, \
     create_policy_from_json_and_return_arn, attach_policy_to_role, \
     remove_policy_being_replaced, tag_role, get_all_role_tags, delete_role_tags, \
-    execute_statement, create_role_and_await_consistency, get_emrfs_roles, remove_user_role
+    execute_statement, create_role_and_await_consistency, get_emrfs_roles, remove_user_role, \
+    get_groups_for_user
```

Comment:
The list of imports is getting a bit out of hand, I'd recommend just doing `import aws_caller` and using the functions like `aws_caller.function_name`